index.markdown

```diff
@@ -1,10 +1,75 @@
 ---
-# Feel free to add content and custom Front Matter to this file.
-# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
-
 layout: default
 ---
 
-# Big Test Energy
+# UWB Hacks 2020 Documentation
+
+![logo](https://cdn.discordapp.com/attachments/652802883531571210/668393411564404736/websiteDesign1.png)
+Temporary logo placeholder using an awesome design created by Erica
+
+Welcome to the documentation page for UWB ACM's annual hackathon! Check out 
+some of the guides below to prepare for the big day, find inspiration
+for potential projects, or get help if you are feeling lost or stuck. 
+
+
+## Schedule
+  
+TODO: This is mostly pulled from last year's schedule
+
+| Time     | Event                           |
+|---------:|:--------------------------------|
+|  9:15 AM | Brekkie time                    |
+| 10:00 AM | Opening Ceremony                |
+| 10:30 AM | Team-Building Mixer             |
+| 11:15 AM | Cloud Computing Workshop #1     |
+| 12:00 PM | Free Lunch Provided             |
+| 12:45 PM | Cloud Computing Workshop #2     |
+|  2:45 PM | Greyhats Workshop?              |
+|  6:00 PM | Free Dinner Provided            |
+|  9:15 PM | Project Presentations           |
+| 10:00 PM | Awards & Closing Ceremony       |
+
+## Submitting Your Project
+
+1. Submit your project to devpost? 
+2. Maybe general outline of steps here on index page, then link to a page with 
+more in-depth instructions under logistics section of useful info
+
+
+## Useful Info
```

Comment:
```suggestion
## Event Info

Maybe move this section before the Submitting Your Project section?
```

Reply:
Yeah, I kept debating with myself on that one! I think you're right - it makes more sense chronologically to have the submission section follow the getting ready/implementation stuff.